keyboards/idobo/keymaps/hyena/config.h

```diff
@@ -0,0 +1,19 @@
+/* Copyright 2018 MechMerlin
```

Comment:
This file can be deleted.

Reply:
info.json was removed.

Reply:
I'm still working on two layers which involve Unicode. I will review the need of `config.h` when I finish the implementation.